docs/faq.md

```diff
@@ -21,9 +21,9 @@ Pocket Network is an open-source decentralized RPC network with a contributor-fr
 
 **DataBuilder Hackathon participants are strongly encouraged to use Pocket Network.**  
 
-All hackathon participants have access to the Pocket Network decentralized RPC endpoints. You can go to the Pocket Portal and sign up for a free account, and you'll be able to access any one of dozens of blockchains for up to 250,000 relays per day at no charge. [https://www.portal.pokt.network/](https://www.portal.pokt.network/) 
+All hackathon participants have access to the Pocket Network decentralized RPC endpoints. You can go to the Pocket Portal and sign up for a free account, and you'll be able to access any one of dozens of blockchains for up to 250,000 relays per day at no charge. [https://www.portal.pokt.network/](https://www.portal.pokt.network/?utm_medium=cta&utm_source=odc&utm_campaign=data-builder-hackathon&utm_content=faq-page) 
 
-Users will also have access to Pocket's robust RPC API, with connections available to Ethereum and all other major blockchains.  More information about the Pocket Network APIs are available here: [https://docs.pokt.network/api-docs/](https://docs.pokt.network/api-docs/)  In order to query againsts these inputs you may choose to use True Blocks or another solution to create an index based on wallet ID.  
+Users will also have access to Pocket's robust RPC API, with connections available to Ethereum and all other major blockchains.  More information about the Pocket Network APIs are available here: [https://docs.pokt.network/api-docs/](https://docs.pokt.network/api-docs/?utm_medium=cta&utm_source=odc&utm_campaign=data-builder-hackathon&utm_content=faq-page)  In order to query againsts these inputs you may choose to use True Blocks or another solution to create an index based on wallet ID.  
```

Comment:
Should this page lead to the API docs page. I tested and it leads to a  blank page... wanted to confirm if we should be using `https://docs.pokt.network/api-docs/pokt/#/?utm_medium=cta&utm_source=odc&utm_campaign=data-builder-hackathon&utm_content=faq-page` instead so that users end up at the exact API docs

Reply:
Updated api doc url